pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_about-assemblies.adoc

```diff
@@ -8,7 +8,7 @@
 = About Assemblies
 
 [role="_abstract"]
-In {ProductName} {ProdctNumber}, each assembly encompasses a single user story. An assembly can include concept, procedure, or reference modules but all modules in an assembly must relate to that specific user story. Previously, many Red Hat documentation teams used assemblies as the structure of user guides and other "books." In {ProductName} {ProductNumber}, there are no books, only user stories, with one assemply per user story. This change is part of our efforts to make it easier for users to search for each specific action they want to perform. 
+In {ProductName} {ProdctNumber}, each assembly encompasses a single user story. An assembly can include concept, procedure, or reference modules but all modules in an assembly must relate to that specific user story. Previously, many Red Hat documentation teams used assemblies as the structure of user guides and other "books." In {ProductName} {ProductNumber}, there are no books, only user stories, with one assembly per user story. This change is part of our efforts to make it easier for users to search for each specific action they want to perform. 
```

Comment:
@RichardHoch 

```{ProductNumber}```

Reply:
1. There are "Guides" in Pantheon 2, which are based on user stories.
2. "...with one assembly per user story."
What about nested assemblies? They will eventually be supported. See https://redhat-documentation.github.io/modular-docs/#what-modular-documentation-is.
3. "This change is part of our efforts to make it easier for users to search for each specific action they want to perform."
Suggestion: "Focusing on user stories is part of the Flexible Customer Content (FCC) effort at Red Hat. One of the goals of this approach is to make it easier..."